library/systemd/src/lib/yast2/systemd_service.rb

```diff
@@ -0,0 +1,189 @@
+require "yast2/systemd_unit"
+
+module Yast2
+  module Systemd
```

Comment:
To be compliant with ruby standards, the name space hierarchy should match the directories structure, so this file should be placed in `yast2/systemd/service.rb`. I would also create a new file `yast2/systemd.rb` with a content as:

```
require "yast2/systemd/unit"
require "yast2/systemd/target"
require "yast2/systemd/service"
require "yast2/systemd/socket"

module Systemd; end
```